api/middleware/contentLengthLimiterMiddleware.go

```diff
@@ -0,0 +1,62 @@
+package middleware
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/multiversx/mx-chain-go/api/shared"
+)
+
+type contentLengthLimiterMiddleware struct {
+	MaxSizeBytes int64
+}
+
+// NewContentLengthLimiterMiddleware will abort all requests that have Content-Length size bigger
+// than the one specified in config.
+func NewContentLengthLimiterMiddleware(maxSizeBytes uint64) (*contentLengthLimiterMiddleware, error) {
+	if maxSizeBytes < 10 {
```

Comment:
can you extract 10 into a `const minSizeBytes = 10` + return something like `return nil, fmt.Errorf("%w, min expected %d, received %d", ErrMaxContentLengthTooSmall, minSizeBytes, maxSizeBytes)`

also adding a test on this case would lead to 100% coverage

Reply:
done.